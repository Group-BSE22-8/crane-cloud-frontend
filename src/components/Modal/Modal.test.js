import React from 'react';

import { shallow } from 'enzyme';

import Modal from '.';
import { bool } from 'prop-types';

describe('Test the Model component', () => { 
    const onClickAway = jest.fn();
    const showModal = false;
    const children = <div>Model</div>
    it('checking for proper rendering', () => {
        const ModalComponent = shallow(<Modal onClickAway={onClickAway} 
            children={children}
            showModal={showModal}/>);
        expect(ModalComponent).toMatchSnapshot();
    })
})