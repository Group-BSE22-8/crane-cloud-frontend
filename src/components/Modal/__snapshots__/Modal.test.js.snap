// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Model component checking for proper rendering 1`] = `ShallowWrapper {}`;
